lib/Core/Site/Values/Content.php

```diff
@@ -143,4 +176,42 @@ private function buildField(array $properties = [])
         $this->fields[$field->fieldDefIdentifier] = $field;
         $this->fieldsById[$field->id] = $field;
     }
+
+    private function getLocations()
```

Comment:
In administration these are sorted by path string, so I've done the same here, in https://github.com/netgen/ezplatform-site-api/commit/5ebead06c230721ac825611f5d8dc702ed9de07c.

I don't want to make this too elaborate, the aim is only to enable doing trivial traversal from the template. But if there's a clear need for that in the future, we might add it anyway.